src/container/RelayRenderer.js

```diff
@@ -123,10 +124,17 @@ class RelayRenderer extends React.Component {
     super(props, context);
     this.mounted = true;
     this.state = this._runQueries(this.props);
+
+    this.props.relayContext.injectBatchingStrategy(
+      ReactDOM.unstable_batchedUpdates
+    );
```

Comment:
Or even better, do not inject `ReactDOM.unstable_batchedUpdates` at all. User can inject that himself if he needs it.